docs/wallets/ledger-wallet.mdx

```diff
@@ -19,4 +19,215 @@ for:
+
+
+### Step 3: Install Solana CLI
+
+There are multiple ways to install the Solana tools on your computer depending on your preferred workflow:
+
+* [Use Solana's Install Tool (Simplest option)](https://docs.solanalabs.com/cli/install#use-solanas-install-tool)
+* [Download Prebuilt Binaries](https://docs.solanalabs.com/cli/install#download-prebuilt-binaries)
+* [Build from Source](https://docs.solanalabs.com/cli/install#build-from-source)
+* [Use Homebrew](https://docs.solanalabs.com/cli/install#use-homebrew)
+
+
+The `spl-token` command-line utility can be used to experiment with SPL tokens. Once you have Rust installed, run:
+
+```
+$ cargo install spl-token-cli
+```
+
+Run `spl-token --help` for a full description of available commands.
+
+You can also install Solana in your OS by following the official installation guide [here](https://docs.solana.com/cli/install-solana-cli-tools)
+
+To use Ledger Nano with Solana CLI, you have to make sure:
+* Ensure the Ledger Live application is closed
+* Plug your Nano into your computer's USB port
+* Enter your pin and start the Solana app on the Nano
+* Ensure the screen reads "Application is ready"
+
+
+### Step 4: Set your Wallet Keypair
+Your Nano supports an arbitrary number of valid wallet addresses and signers. To view any address, use the `solana-keygen pubkey` command
+
+Having several wallet addresses can be beneficial for transferring tokens across your personal accounts for various reasons, or employing distinct keypairs on the device to act as signing authorities for a stake account, for instance.
+
+```
+solana-keygen pubkey usb://ledger
+solana-keygen pubkey usb://ledger?key=0
+solana-keygen pubkey usb://ledger?key=1
+```
+
+### Step 5: Send HNT from Ledger
+
+To transfer tokens from an address associated with your Ledger Hardware wallet, you'll need to sign the transaction using the device, utilizing the same keypair URL that was used to generate the address. Ensure your Nano is connected, unlocked using your PIN, that Ledger Live is closed, and that the Solana App is activated on your device, displaying "Application is Ready."
+
+The `spl-token transfer` command allows you to define the recipient address, the amount of tokens to send, and includes the `--keypair` argument to identify the sending keypair. This keypair will sign off on the transaction, resulting in a deduction from the balance of the specified address.
+
+The mint address for HNT is `hntyVP6YFm1Hg25TN9WGLqM12b8TQmcknKrdu1oxWux` on the Solana blockchain.
+
+```
+~$ spl-token transfer hntyVP6YFm1Hg25TN9WGLqM12b8TQmcknKrdu1oxWux <AMOUNT> <RECEIVER> --keypair usb://ledger?key=1
+
+```
```

Comment:
There should be more docs for https://github.com/helium/helium-wallet-rs

Reply:
Although I'm not sure, @madninja does that currently support ledger?

Reply:
@ChewingGlass  Hello! I checked the wallet binary and it seems not support Ledger now.